dataplane/linux/vxlan_mgr.go

```diff
@@ -194,16 +260,44 @@ func (m *vxlanManager) CompleteDeferredWork() error {
 				continue
 			}
 
-			routes = append(routes, routetable.Target{
-				Type: routetable.TargetTypeVXLAN,
-				CIDR: cidr,
-				GW:   ip.FromString(vtep.Ipv4Addr),
-			})
+			if r.Type == proto.RouteType_DEFAULT {
+				defaultRoute := routetable.Target{
+					Type: routetable.TargetTypeVXLANParent,
+					CIDR: cidr,
+					GW:   ip.FromString(r.Gw),
+				}
+
+				parentRoutes = append(parentRoutes, defaultRoute)
+				logCtx.WithField("route", r).Debug("adding default route to list for addition")
+			} else {
+				vxlanRoute := routetable.Target{
+					Type: routetable.TargetTypeVXLAN,
+					CIDR: cidr,
+					GW:   ip.FromString(vtep.Ipv4Addr),
+				}
+
+				vxlanRoutes = append(vxlanRoutes, vxlanRoute)
+				logCtx.WithField("route", vxlanRoute).Debug("adding vxlan route to list for addition")
+			}
+		}
+
+		logrus.WithField("vxlanroutes", vxlanRoutes).Debug("VXLAN manager sending VXLAN L3 updates")
+		m.routeTable.SetRoutes(m.vxlanDevice, vxlanRoutes)
+
+		if m.parentRouteTable != nil {
+			if parentDevice, err := m.getLocalVTEPParent(); err == nil {
+				ifName := parentDevice.Attrs().Name
+				log.WithField("link", parentDevice).WithField("routes", parentRoutes).Debug("setting default routes")
```

Comment:
I think it would be nice to make this log symmetric with the "VXLAN manager sending VXLAN L3 updates" one above.

e.g. "VXLAN manager sending unencapsulated L3 updates"